scripts/hid_configurator/NrfHidDevice.py

```diff
@@ -485,3 +485,9 @@ def config_get(self, module_name, option_name, poll_interval=POLL_INTERVAL_DEFAU
 
     def config_set(self, module_name, option_name, value, poll_interval=POLL_INTERVAL_DEFAULT):
         return self._config_operation(module_name, option_name, False, value, poll_interval)
+
+    def get_dfu_module_name(self):
```

Comment:
can it be generic? maybe it will be useful for some other place
e.g.
```
get_module_name_by_root(self, name):
   for ket in self.dev_config:
      if '{}/'.format(name) in key:
         return key
   return None
```